Tools/gdb/libpython.py

```diff
@@ -1492,18 +1492,18 @@ def get_index(self):
     #   - everything else
 
     def is_python_frame(self):
-        '''Is this a PyEval_EvalFrameEx frame, or some other important
+        '''Is this a _PyEval_EvalFrameDefault frame, or some other important
         frame? (see is_other_python_frame for what "important" means in this
         context)'''
-        if self.is_evalframeex():
+        if self.is_evalframedefault():
             return True
         if self.is_other_python_frame():
             return True
         return False
 
-    def is_evalframeex(self):
-        '''Is this a PyEval_EvalFrameEx frame?'''
-        if self._gdbframe.name() == 'PyEval_EvalFrameEx':
+    def is_evalframedefault(self):
+        '''Is this a _PyEval_EvalFrameDefault frame?'''
+        if self._gdbframe.name() == '_PyEval_EvalFrameDefault':
```

Comment:
Please put '_PyEval_EvalFrameDefault' in a constant at the top of the file, so if someone really needs to catch PyEval_EvalFrameEx(), it will be easier to update python-gdb at runtime.